source/client-side-encryption/client-side-encryption.rst

```diff
@@ -286,13 +289,45 @@ Data keys are stored as documents in a special MongoDB collection. Data
 keys are protected with encryption by a KMS provider (AWS KMS, Azure key
 vault, GCP KMS, or a local master key).
 
+metadataClient
```

Comment:
This deserves a minor version bump.  Also, it's a big enough change that at least one other driver should implement before we push this to master

Reply:
Oh good point, I bumped the version to 1.3.0.

I agree, I'll hold off merging until another driver implements this. It was a small change in Go (mostly following suit with how the `keyVaultClient` is handled).